setup.py

```diff
@@ -20,6 +20,8 @@
     install_requires=[
         "requests>=2.20.1",
         "XlsxWriter>=1.2.1",
-        "openpyxl>=3.0.6"
-    ]
+        "openpyxl>=3.0.6",
+        "python-dateutil>=2.8.1"
+    ],
+    scripts=['bin/cyberwatch-cli']
```

Comment:
Du peut ajouter de la documentation dans le README.md

Reply:
Documentation was added to `docs/cli.md`.